src/app/components/shell/microsurvey-popup/adoption-content.js

```diff
@@ -0,0 +1,72 @@
+import React, {useState, useEffect, useRef} from 'react';
+import {accountsModel} from '~/models/usermodel';
+import {fetchFromCMS} from '~/helpers/controller/cms-mixin';
+import {cmsPost} from '~/models/cmsFetch';
+
+function useAccountNameAndId() {
+    const [value, setValue] = useState([]);
+
+    useEffect(() => {
+        accountsModel.load().then((response) => {
+            setValue([response.first_name, response.id]);
+        });
+    }, []);
+
+    return value;
+}
+
+function useAdoptions(accountId) {
+    const [adoptions, setAdoptions] = useState([]);
+
+    useEffect(() => {
+        if (accountId) {
+            fetchFromCMS(`salesforce/renewal/${accountId}/`).then(setAdoptions);
+        }
+    }, [accountId]);
+
+    return adoptions;
+}
+
+function BookCheckbox({info}) {
+    return (
+        <div className="control-group">
+            <label>
+                <input type="checkbox" name={info.id} value={info.yearly_students} />
+                {info.book_name}
+            </label>
+        </div>
+    );
+}
+
+async function submitForm(formRef, accountId) {
+    for (const cb of formRef.current.querySelectorAll('[type=checkbox]')) {
+        const students = cb.checked ? cb.value : 0;
+        const postOutput = await cmsPost(`salesforce/renewal/${accountId}/`, {
+            // eslint-disable-next-line camelcase
+            confirmed_yearly_students: students,
```

Comment:
if you hit this a lot, `eslint` has a `properties` config that will ignore property keys https://eslint.org/docs/rules/camelcase